_sources/Conditionals/intro-TurtlesandConditionals.rst

```diff
@@ -0,0 +1,92 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: condition-1-
+   :start: 1
+
+Intro: What we can do with Turtles and Conditionals
+---------------------------------------------------
+
+To build further on our turtle drawings, we can start to include something called a **selection** or 
+**conditional statement**. Using this kind of statement will allow up to check conditions and change the 
+behavior of the program accordingly. 
+
+.. activecode:: ac7_1_1
+
+    import turtle
+    wn = turtle.Screen()
+
+    Amy = turtle.Turtle()
```

Comment:
Fix capitalization -- I get that the names are proper nouns, but I am a stickler on following the convention that variables should be lower case while classes are Capitalized.